omnibus-base-sepolia-andromeda.toml

```diff
@@ -10,12 +10,10 @@ include = [
     "tomls/pools/spartan-council.toml",                             # Create Spartan Council Pool
     "tomls/markets/spot-factory.toml",
     "tomls/markets/perps-highcap-factory.toml",                     # First supermarket (High MCap)
-    "tomls/markets/perps-lowcap-factory.toml",                      # Second supermarket (Low MCap)
```

Comment:
Just a note here that removing it does not mean it will be "undeployed" so this perps market still exists in the system

Reply:
thanks for noticing!

Reply:
added it back with a comment that is DISABLED